ivy_tests/test_ivy/conftest.py

```diff
@@ -39,14 +39,12 @@
 
 def pytest_configure(config):
     global available_frameworks
-    # traceback
-    raw_value = config.getoption("--tb-mode")
-    set_exception_trace_mode(raw_value)
-    raw_value = config.getoption("--tb")
-    if raw_value == "auto":
-        config.option.tbstyle = "line"
-    else:
-        config.option.tbstyle = raw_value
+
+    # Ivy Exception traceback
+    set_exception_trace_mode(config.getoption("--ivy-tb"))
+
+    # Pytest traceback
+    config.option.tbstyle = config.getoption("--tb")
```

Comment:
Hey @CatB1t! If we're going to keep the default as 'full', this line is not required. The default tracemode for Pytest is "auto" which displays the full traceback. I had actually changed the default behavior to "line" which displays a shorter, more concise traceback.